test/test.js

```diff
@@ -343,6 +358,27 @@ test('--output with invalid input outputs violations', async t => {
   }
 });
 
+test('--output with invalid input and --json outputs issues as json', async t => {
+  const input = [
+    'Heading',
+    '',
+    'Text ',
+    ''
+  ].join('\n');
+  const output = '../outputC.json';
```

Comment:
Unique file name or this will fail randomly

Reply:
This filename is unique - .json extension. But I changed it to outputJsonC.json to make it more explicit.